app/Repositories/ChatRepository.php

```diff
@@ -313,9 +313,9 @@ public function systemMessage($message, $bot = null)
         if ($bot) {
             $this->message($systemUserId, $this->systemChatroom(), $message, null, $bot);
         } else {
-            $systemBotId = Bot::where('slug', 'systembot')->first()->id;
+            $systemBotId = Bot::where('command', 'systembot')->first()->id;
```

Comment:
Why this change?

Reply:
Because it appears that in some earlier version of DB seeder or something, a slug with just "system" was created, with "command" systembot. Hence there was no bot named "Systembot", causing all chat announces to error. However, I notice at least the current seeder does slug "systembot", so might have been just my mixup or something. Using command instead of slug fixed this issue, but I guess it can just as well be fixed by editing the slug in the DB since that's what it should be. Hence I've removed this change from this commit. My bad.